helm/vertical-pod-autoscaler-app/templates/crd-patch/crd-job.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
```

Comment:
I think this job is useless now

Reply:
The crd ownership should have been changed to the crd app now

Reply:
Should I delete it ?

Reply:
I would think so yes see https://github.com/giantswarm/roadmap/issues/2009

Reply:
Ok then let's delete it :)

Reply:
Done